### PR TITLE
feat: added KAVA kavapool

### PR DESCRIPTION
**Pull request by snowwhitedev**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below if the PR is for listing a new protocol
3. If you want to update the protocol info you entered in the form below, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI

---

##### Twitter Link:
- https://twitter.com/unoreinsure

##### List of audit links if any:
- https://beosin.com/audits/UnoRe_202201251340.pdf
- https://solidity.finance/audits/CoverPortal/
- https://solidity.finance/audits/SSRP/

##### Website Link:
- https://www.unore.io/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
- https://s2.coinmarketcap.com/static/img/coins/64x64/8875.png

##### Chain:
- Ethereum
- BSC
- KAVA

##### Short Description (to be shown on DefiLlama):
Uno Re is a DeFi Insurance platform aiming to safeguard users' assets from unforeseen circumstances. With the launch of its Cover Portal, users can easily insure their staked assets and get customized insurance bundle covers.

Insure your assets now at app.unore.io

##### Token address and ticker if any:
- ETH: 0x474021845c4643113458ea4414bdb7fb74a01a77
- BSC: 0x474021845c4643113458ea4414bdb7fb74a01a77

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL is calculated by summing up all amounts which is staked in RiskPools in both ETH, BSC and KAVA

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/unore exports TVL: 
        
 
 
    bsc                       501.50 k
    ethereum                  902.59 k
    kava                      1.8
    
    total                    1.40 M

File: projects/unore/index.js [hunk 1]
```diff
@@ -25,6 +26,18 @@ const ethRiskPools = [
 ]
 const ethSSIPEth = '0x29B4b8674D93b36Bf651d0b86A8e5bE3c378aCF4'
 
+// KAVA pools
+// tokens
+const kavaTokens = [
+  // '0xfA9343C3897324496A05fC75abeD6bAC29f8A40f', // USDC
```
Comment (g1nt0ki) [inline projects/unore/index.js:32]: shouldn't this list be enabled?

File: projects/unore/index.js [hunk 2]
```diff
@@ -25,6 +26,18 @@ const ethRiskPools = [
...
+]
+// risk pools
+const kavaRiskPools = [
+  '0x6cEC77829F474b56c327655f3281739De112B019'
```
Comment (g1nt0ki) [inline projects/unore/index.js:36]: is this address correct? I dont see anything there: https://explorer.kava.io/address/0x6cEC77829F474b56c327655f3281739De112B019/transactions

File: projects/unore/index.js (hunk 1 above)
Comment (snowwhitedev) [reply projects/unore/index.js:32]: This is USDC address which KAVA team provided us, but this USDC coin is not recognized when I tried to calculate tvl,
So I would like to keep it now if possible, so we can remember and update without missing a bit later.

File: projects/unore/index.js (hunk 2 above)
Comment (snowwhitedev) [reply projects/unore/index.js:36]: Yes, correct. This is USDC pool but we have no any stakes there.

**Comment by snowwhitedev** [comment]
Thank you for your good catch and support. @g1nt0ki 
I added my comment.

Regards

File: projects/unore/index.js (hunk 1 above)
Comment (g1nt0ki) [reply projects/unore/index.js:32]: just confirming, I can fix it

**Pull request merged**